stack-citationapi/recipes/deploy-citation-formatting-api.rb

```diff
@@ -0,0 +1,30 @@
+node['deploy'].each do |application, deploy|
+  case application
+  when 'citation_apis'
```

Comment:
This can be removed - you are handling it in `stack-citationapi/recipes/deploy-citation-data-api.rb` already